docs/post-installation/README.md

```diff
@@ -84,7 +95,7 @@ yay -S linux-apfs-dkms-git
 ### General Fonts
 🗛 Those are some essential font packages
 ```bash
-yay -S adobe-source-sans-pro-fonts ttf-dejavu ttf-opensans noto-fonts freetype2 terminus-font ttf-bitstream-vera ttf-dejavu ttf-droid ttf-fira-mono ttf-fira-sans ttf-freefont ttf-inconsolata ttf-liberation libertinus-font
```

Comment:
I would keep the small list of fonts and add `all-repository-fonts` as alternative.

Reply:
Some packages in your list are renamed so they're invalid and every of your package is included in all-repository-fonts (like the name says)

Reply:
The current command works fine for me. As I said, I would keep the small selection, because maybe not everyone wants to install all the fonts.